packages/block-editor/src/components/block-list/zoom-out-separator.js

```diff
@@ -103,7 +104,19 @@ export function ZoomOutSeparator( {
 					data-is-insertion-point="true"
 					onDragOver={ () => setIsDraggedOver( true ) }
 					onDragLeave={ () => setIsDraggedOver( false ) }
-				></motion.div>
+				>
+					<motion.div
+						initial={ { opacity: 0 } }
+						animate={ { opacity: 1 } }
+						exit={ { opacity: 0 } }
+						transition={ {
+							type: 'tween',
+							duration: 0.1,
+						} }
+					>
+						{ __( 'Drop pattern.' ) }
```

Comment:
@jasmussen @draganescu Shall we run with this and then iterate? We have until RC 1 (October 22, 2024) to change this prior to the hard string freeze.

Reply:
Fine from me, assuming we can polish up the little things.

Reply:
I'm going to ask folks working on Zoom Out to start testing everything on TT5 👍 

I'll raise an Issue to track the width thing.

Reply:
What is wrong with two labels?